pkg/ddc/jindo/transform.go

```diff
@@ -401,29 +404,34 @@ func (e *JindoEngine) transformFuseArg(runtime *datav1alpha1.JindoRuntime, datas
 	return []string{baseArg}
 }
 
-func (e *JindoEngine) parseSmartDataImage() (image, tag string) {
+func (e *JindoEngine) parseSmartDataImage() (image, tag, dnsServer string) {
```

Comment:
parsing dnsserver in `func parseSmartDataImage()`  is misleading. We can rename this function into `func getSmartDataConfigurations`?

Reply:
Done